src/pages/resultsView/ResultsViewPageStore.ts

```diff
@@ -106,6 +109,21 @@ export class ResultsViewPageStore {
         invoke: async () => fetchSamples(this.sampleIds, this.studyId)
     }, []);
 
+    readonly samplesWithoutCancerTypeClinicalData = remoteData({
+        await: () => [
+            this.sampleIds,
+            this.clinicalDataForSamples
+        ],
+        invoke: async () => fetchSamplesWithoutCancerTypeClinicalData(this.sampleIds, this.studyId, this.clinicalDataForSamples)
```

Comment:
not for now, but we will need to redesign it when we want to support multiple studies